src/styles/styles.css

```diff
@@ -21,14 +20,31 @@ body {
 .mobile-nav {
   display: block;
 }
+.mobile-nav-button {
+  border: none;
+  background: rgba(255, 255, 255, 0.7);
+  margin: 10px;
+  padding: 10px;
+}
 
 .profile-wrapper {
+  background: rgba(255, 255, 255, 0.6);
   display: none;
+  overflow: auto;
+  padding: 10px;
 }
 .profile-wrapper.show {
   display: block;
 }
 
+.player-list {
+  height: 500px;
```

Comment:
that won't work on mobile landscape
please try to relative units (%) and test it's behaviour on mobile

Reply:
Changed to %.